chore(docs): introduce helm deploy method in use kubernetes section (#1793)
yuzutech/kroki: docs/modules/setup/pages/use-kubernetes.adoc

Read more about how to publish a service on the {uri-k8s-docs-publish-service}[Kubernetes documentation] website.
====

> == Deploy use Kubernetes object definition files

Reviewer: I think it should be:

```suggestion
== Deploy using Kubernetes object definition files
```

Author: revised

You can then create the Kubernetes objects as a batch or individually, similar to what was shown previously.

Validation and uninstall/removal of Kubernetes objects can be followed using the exact same commands as shown previously too.

> == Deploy use Helm Charts

Reviewer: ```suggestion
== Deploy using Helm Charts
```

Author: revised as suggested, thank!

> You can alternatively deploy Kroki using Helm Chart provided link:https://github.com/cowboysysop/charts/tree/master/charts/kroki[`here`]. It's much easier to deploy Kroki with charts, since all object definition files are managed by helm, in a style like App. 

Reviewer: ```suggestion
You can alternatively deploy Kroki using https://github.com/cowboysysop/charts/tree/master/charts/kroki[Helm Charts]. It's much easier to deploy Kroki with charts, since all object definition files are managed by Helm. 
```

I don't understand the "in a style like App." ?
Also, could you please declare `https://github.com/cowboysysop/charts/tree/master/charts/kroki` as an attribute (at the top of this file):

```
:uri-helm-charts: https://github.com/cowboysysop/charts/tree/master/charts/kroki
```

And then:

```
You can alternatively deploy Kroki using {uri-helm-charts}[Helm Charts]. 
```

Author: For the text `in a style like App.`, I meant to say that install a helm chart is like install an application. Users does not need to dig into the objects/resources inside a chart. 

I removed the text `in a style like App.` here, so that not causing ambiguity.

Also, attribute is added as suggested.

> [NOTE]
> ====
> This is an unofficial chart maintained by `cowboysysop`. 
> ====

Reviewer: ```suggestion
NOTE: This is an unofficial chart maintained by `cowboysysop`. 
```

To install Kroki with release name `my-kroki`, use the following command:

.Example command with `helm` installed
[source,cli]
----
helm repo add cowboysysop https://cowboysysop.github.io/charts/
helm install my-kroki cowboysysop/kroki
----

> Validation of Kubernetes objects can be followed using the exact same commands as shown previously too. Or, you can use the following helm command to verify the installation.

Reviewer: > Validation of Kubernetes objects can be followed using the exact same commands as shown previously too. 

How? Do I need to pass an extra command-line argument? --validate?

Author: I have added command-line argument `--show-resources --show-desc`, to let helm show the detailed resources status. The detailed status can be used to verify a successful installation. The example output of `helm status` command is updated as well.

You can then create the Kubernetes objects as a batch or individually, similar to what was shown previously.

Validation and uninstall/removal of Kubernetes objects can be followed using the exact same commands as shown previously too.

== Deploy using Helm Charts

> You can alternatively deploy Kroki using {uri-helm-charts}[Helm Charts]. It's much easier to deploy Kroki with charts, since all object definition files are managed by helm.

Reviewer: ```suggestion
You can alternatively deploy Kroki using {uri-helm-charts}[Helm Charts]. It's much easier to deploy Kroki with charts, since all object definition files are managed by Helm.
```

NOTE: This is an unofficial chart maintained by `cowboysysop`. 

> To install Kroki with release name `my-kroki`, use the following command:

Reviewer: ```suggestion
To install Kroki using the name `my-kroki`, use the following command:
```

.Example command with `helm` installed
[source,cli]
----
helm repo add cowboysysop https://cowboysysop.github.io/charts/
helm install my-kroki cowboysysop/kroki
----

> Validation of Kubernetes objects can be followed using the exact same commands as shown in <<Validate that Kroki is running>>. Or, you can use the following helm command to verify the installation.

Reviewer: ```suggestion
Use the following helm command to verify the installation:
```

.Example command with `helm` installed
[source,cli]
----
helm status my-kroki --show-resources --show-desc
----

If the installation is successful, the following status is shown:

....
NAME: my-kroki
LAST DEPLOYED: Mon Oct  7 17:06:09 2024
NAMESPACE: default
STATUS: deployed
REVISION: 1
DESCRIPTION: Install complete
RESOURCES:
==> v1/Pod(related)
NAME                     READY   STATUS        RESTARTS   AGE
kroki-6db8878b8f-zn8kz   5/5     Running       0          13d

==> v1/ServiceAccount
NAME    SECRETS   AGE
kroki   0         14d

==> v1/Service
NAME    TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
kroki   ClusterIP   10.43.100.80   <none>        8000/TCP   14d

==> v1/Deployment
NAME    READY   UP-TO-DATE   AVAILABLE   AGE
kroki   1/1     1            1           14d

NOTES:
1. Get the application URL by running these commands:
  export POD_NAME=$(kubectl get pods --namespace default -l "app.kubernetes.io/name=kroki,app.kubernetes.io/instance=kroki" -o jsonpath="{.items[0].metadata.name}")
  echo "Visit http://127.0.0.1:8000/ to use your application"
  kubectl --namespace default port-forward $POD_NAME 8000:8000
....

> To uninstall Kroki, use the command `helm uninstall` as follows:

Reviewer: ```suggestion
You can use the exact same commands as shown in <<Validate that Kroki is running>> to make sure that Kroki is running fine.

To uninstall Kroki, use the command `helm uninstall` as follows:
```
